sdks/python/apache_beam/io/avroio.py

```diff
@@ -39,7 +39,7 @@
 class ReadFromAvro(PTransform):
   """A ``PTransform`` for reading avro files."""
 
-  def __init__(self, file_pattern=None, min_bundle_size=0, validate=True):
+  def __init__(self, file_pattern=None, min_bundle_size=0, validate=True):  # pylint: disable=super-on-old-class
```

Comment:
I have no idea why this is failing. This PTransform doesn't seem any different from the others :/